installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+
+[id="installing-preparing-existing-components"]
+=== How many existing components do you want to use with your cluster?
+
+Because of the way that the operating system and platform intertwine, it is far easier to complete an installation where you let the installation program for {product-title} stand up all of the infrastructure. These are called _installer provisioned infrastructure_ installations. In this type of installation, you can provide some existing infrastructure to the cluster, but the installation program deploys all of the machines that your cluster initially needs.
+
+You can deploy an installer-provisioned infrastructure cluster without specifying any customization to the cluster or its underlying machines to xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[AWS], xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[Azure], or xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[GCP]. These installation methods are the fastest way to deploy a production-capable {product-title} cluster.
+
+If you need to perform basic configuration for your installer-provisioned infrastructure cluster, such as the region in your cloud that you deploy the machines to and the instance type for the cluster machines, you can customize an installation for xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[AWS], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Azure], or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[GCP].
+
+For installer-provisioned infrastructure installations, you can use an existing xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[VPC in AWS], xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[vNet in Azure], or xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[VPC in GCP]. You can also reuse part of your networking so that your cluster in xref:../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[AWS], xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[Azure], or xref:../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[GCP] can coexist with existing IP address allocations in your environment and integrate with existing MTU and VXLAN configurations. If you have existing accounts and credentials on these clouds, you can modify the accounts to have the required permissions to install {product-title} clusters on them.
```

Comment:
```
If you have existing accounts and credentials on these clouds, you can modify the accounts to have the required permissions to install {product-title} clusters on them.
```
This sentence does not seem to fit into this section. This section is about custom configurations that you user can do while still doing an installer-provisioned installation. This sentence is more about what the user must do to configure their account. Regardless of whether the user has an existing account or is creating a new account, the user still needs to configure the required permissions for the account. Maybe what you want to say is something like the following.
```
If you have existing accounts and credentials on these clouds, you can re-use them, but you may need to modify the accounts to have the required permissions to install {product-title} clusters on them.
```